full/Dockerfile

```diff
@@ -112,8 +112,8 @@ RUN curl -fsSL https://storage.googleapis.com/golang/go$GO_VERSION.linux-amd64.t
     rm -rf $GOPATH/src && \
     rm -rf $GOPATH/pkg
 # user Go packages
-ENV GOPATH=/workspace \
-    PATH=/workspace/bin:$PATH
+ENV GOPATH=/workspace/go \
+    PATH=$GOPATH/bin:$PATH
```

Comment:
Won't `$GOPATH` be `$HOME/go` at this point?

Reply:
Good point - fixed it